jsonrpc/personal_endpoint.go

```diff
@@ -0,0 +1,126 @@
+package jsonrpc
+
+import (
+	"errors"
+	"fmt"
+	"math"
+	"time"
+
+	"github.com/0xPolygon/polygon-edge/accounts"
+	"github.com/0xPolygon/polygon-edge/accounts/keystore"
+	"github.com/0xPolygon/polygon-edge/crypto"
+	"github.com/0xPolygon/polygon-edge/types"
+)
+
+type Personal struct {
+	accManager accounts.BackendManager
+}
+
+func NewPersonal(manager accounts.BackendManager) *Personal {
+	return &Personal{accManager: manager}
+}
+
+func (p *Personal) ListAccounts() ([]types.Address, Error) {
+	return p.accManager.Accounts(), nil
+}
+
+func (p *Personal) NewAccount(password string) (types.Address, error) {
+	ks, err := getKeystore(p.accManager)
+	if err != nil {
+		return types.ZeroAddress, err
+	}
+
+	acc, err := ks.NewAccount(password)
+	if err != nil {
+		return types.ZeroAddress, fmt.Errorf("can't create new account")
+	}
+
+	return acc.Address, nil
+}
+
+func (p *Personal) UpdatePassphrase(addr types.Address, oldPassphrase, newPassphrase string) (bool, error) {
+	ks, err := getKeystore(p.accManager)
+	if err != nil {
+		return false, err
+	}
+
+	err = ks.Update(accounts.Account{Address: addr}, oldPassphrase, newPassphrase)
+	if err != nil {
+		return false, err
+	}
+
+	return true, nil
+}
+
+func (p *Personal) ImportRawKey(privKey string, password string) (types.Address, error) {
+	key, err := crypto.HexToECDSA(privKey)
+	if err != nil {
+		return types.ZeroAddress, err
+	}
+
+	ks, err := getKeystore(p.accManager)
+	if err != nil {
+		return types.ZeroAddress, err
+	}
+
+	acc, err := ks.ImportECDSA(key, password)
+
+	return acc.Address, err
+}
+
+func (p *Personal) UnlockAccount(addr types.Address, password string, duration uint64) (bool, error) {
+	const max = uint64(time.Duration(math.MaxInt64) / time.Second)
```

Comment:
I am not sure this expression is ok. time.Second = 1.000.000.000 expressed as an int64. so const max should be 9.223.372.036, it might be better to set max to some const value which we consider as a max duration value.  That way we will know what max duration is in seconds, this way we have to calculate first value of that expression and in addition 9 billions of seconds seems too large.